scripts/drone/events/release.star

```diff
@@ -266,7 +266,7 @@ def enterprise_pipelines(ver_mode = ver_mode, trigger = release_trigger):
 
     environment = {"EDITION": "enterprise"}
 
-    services = integration_test_services(edition = "enterprise")
+    services = integration_test_services()
```

Comment:
Since now *all* services are launched, does it also make sense to run the redis/memcached integration tests for non-enterprise builds (line 338)?

Reply:
not sure I understand: do you mean since in enterprise builds will run also the OSS ones we don't need to run them for non-enterprise builds?

Reply:
I think Horst is suggesting that redis/memcached tests might not be running for non-enterprise builds. For context, when I added them, enterprise was the only one that had redis/memcached integration tests.

Reply:
thank for the clarification: I have addeed them also in the OSS builds